src/pages/Homepage/HomepageStatictics.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+	<div class="small-10 small-offset-1 large-10 large-offset-1 columns stats-box">
+		<div class="row">
+			<!-- loans funded -->
+			<div class="small-6 large-2 large-offset-1 columns stats-box__group even">
+				<div class="stats-box__stat">
+					${{ amountFunded | numeral('0.0a') }}
+				</div>
+				<div class="stats-box__stat-label">
+					loans funded
+				</div>
+			</div>
+			<!-- # lenders  -->
+			<div class="small-6 large-2 columns stats-box__group odd">
+				<div class="stats-box__stat">
+					{{ numLenders | numeral('0.0a') }}
+				</div>
+				<div class="stats-box__stat-label">
+					lenders
+				</div>
+			</div>
+			<!-- repayment rate -->
+			<div class="small-6 large-2 columns stats-box__group even">
+				<div class="stats-box__stat">
+					{{ repaymentRate | numeral('0%') }}
+				</div>
+				<div class="stats-box__stat-label">
+					repayment rate
+				</div>
+			</div>
+			<!-- # countries  -->
+			<div class="small-6 large-2 columns stats-box__group odd">
+				<div class="stats-box__stat">
+					{{ numCountries }}
+				</div>
+				<div class="stats-box__stat-label">
+					countries
+				</div>
+			</div>
+			<!-- # years -->
+			<div class="small-12 large-2 columns">
+				<div class="stats-box__stat">
+					{{ yearsSinceStartCalc }}
+				</div>
+				<div class="stats-box__stat-label">
+					years
+				</div>
+			</div>
+		</div>
+	</div>
+</template>
+
+<script>
+import _get from 'lodash/get';
+import { differenceInYears } from 'date-fns';
+import homepageStatistics from '@/graphql/query/homepage/homepageStatistics.graphql';
+
+export default {
+	inject: ['apollo'],
+	apollo: {
+		query: homepageStatistics,
+		preFetch: true,
+		result({ data }) {
+			this.amountFunded = _get(data, 'general.kivaStats.amountFunded');
```

Comment:
You'll need to create the reactive data object (`data() { return { your: defaults },`) and instantiate each of these data points there. Give them each a default value based on what is shown in prod currently... Then you won't have to worry about any unforeseen failure in the prefetch.